ui/src/views/Denylist/ViewDenylistAddress/index.jsx

```diff
@@ -0,0 +1,108 @@
+import { hot } from 'react-hot-loader';
+import React, { Component, Fragment } from 'react';
+import { graphql, withApollo } from 'react-apollo';
+import Spinner from '@mozilla-frontend-infra/components/Spinner';
+import Dashboard from '../../../components/Dashboard';
+import DenylistForm from '../../../components/DenylistForm';
+import ErrorPanel from '../../../components/ErrorPanel';
+import formatError from '../../../utils/formatError';
+import denylistAddressQuery from './denylistAddress.graphql';
+import addAddressQuery from './addAddress.graphql';
+import deleteAddressQuery from './deleteAddress.graphql';
+
+@hot(module)
+@withApollo
+@graphql(denylistAddressQuery, {
+  skip: ({ match: { params } }) => !params.notificationAddress,
+  options: ({ match: { params } }) => ({
+    variables: {
+      filter: {
+        notificationAddress: decodeURIComponent(params.notificationAddress),
+      },
+    },
+  }),
+})
+export default class ViewDenylistAddress extends Component {
+  state = {
+    loading: false,
+    // Mutation errors
+    error: null,
+  };
+
+  handleDeleteAddress = async (notificationType, notificationAddress) => {
+    this.setState({ error: null, loading: true });
+
+    try {
+      await this.props.client.mutate({
+        mutation: deleteAddressQuery,
+        variables: {
+          address: {
+            notificationAddress,
+            notificationType,
+          },
+        },
+      });
+
+      this.setState({ error: null, loading: false });
+
+      this.props.history.push(`/notify/denylist`);
+    } catch (error) {
+      this.setState({ error, loading: false });
+    }
+  };
+
+  handleAddAddress = async (notificationType, notificationAddress) => {
+    this.setState({ error: null, loading: true });
+
+    try {
+      await this.props.client.mutate({
+        mutation: addAddressQuery,
+        variables: {
+          address: {
+            notificationAddress,
+            notificationType,
+          },
+        },
+      });
+
+      this.setState({ error: null, loading: false });
+
+      const url = `/notify/denylist/${encodeURIComponent(notificationAddress)}`;
```

Comment:
No need to store the `url` in a temporary variable since it's not used in one place.